portfolio/src/main/webapp/script.js

```diff
@@ -18,11 +18,21 @@
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const hobbies = 
+      ['Running', 'Hiking', 'Rock Climbing', 'Photography', 'Traveling'];
+  const links =
+      ['Projects', 'Hobbies'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random link.
+  const link = links[Math.floor(Math.random() * links.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  if (link == 'Hobbies'){
+      greetingContainer.innerHTML = "<a href='hobbies.html'> Hobbies </a>";
+  }
+  else {
+      greetingContainer.innerHTML = "<a href='projects.html'> Projects </a>";
+  }
```

Comment:
nit: you could alternatively just put these strings in your links array, to avoid maintaining a separate mapping

```js
const links =
      ['<a href="hobbies.html"> Hobbies </a>', '<a href="projects.html"> Projects </a>'];
```